tests/unit/test_concurrency.c

```diff
@@ -75,7 +75,11 @@ SENTRY_TEST(concurrent_init)
 {
     long called = 0;
 
-#define THREADS_NUM 100
+#ifdef SENTRY_PLATFORM_NX
+#    define THREADS_NUM 90
```

Comment:
This is fine, but I am curious to understand the difference.

Reply:
There's a limit on the number of threads and our original tests triggered it. We can either lower the number for all tests or do it as above with ifdef

Reply:
Okay, that makes sense. Let's go ahead and leave it like this.